lib/config.js

```diff
@@ -135,7 +135,7 @@ class Local extends Config {
     }
 
     getCollection($id) {
-        if (!this.has("collection")) {
+        if (!this.has("collections")) {
```

Comment:
Feels like breaking change for appwrite.json

Reply:
I think its okay as its a major version